src/express/router.ts

```diff
@@ -1,16 +1,22 @@
-import { Router } from 'express';
+import { NextFunction, Response, Router } from 'express';
 import { StatusCodes } from 'http-status-codes';
 import { shragaCallbackMiddleware, shragaLoginMiddleware } from './auth';
 import { ServerError } from './error';
-import featuresRouter from './incomingFile/router';
+import outgoingRouter from './outgoing/router';
+import userRouter from './users/router';
 
-const appRouter = Router();
+const appRouter: Router = Router();
```

Comment:
Type not needed. (infered from Router();)
```suggestion
const appRouter = Router();
```

Reply:
done